python/templates/Constructor.jl.jinja2

```diff
@@ -0,0 +1,26 @@
+include("{{ class.bare_type }}Struct.jl")
+{% for include in includes_jl['constructor'] %}
+{{ include }}
+{% endfor %}
+function {{ class.bare_type }}()
+	return {{ class.bare_type }}{% if params_jl %}{ {% for relation in params_jl %} {{ relation }}, {% endfor %} }{% endif %}(
```

Comment:
```suggestion
	return {{ class.bare_type }}{% if params_jl %}{ {{ params_jl | join(',') }} }{% endif %}(
```
I just realized that this can probably be simplified like suggested. This makes this template slightly more readable and has the additional advantage that we are not left with a trailing comma.

We are already using it in a few places, e.g. here, where we even pick out an attribute for each element of the list:
https://github.com/AIDASoft/podio/blob/a8efda986f776892e90e7aafa4958aaf18a1e6c7/python/templates/macros/declarations.jinja2#L62